src/v3/test/integration/__snapshots__/authenticator-verification-webauthn.test.tsx.snap

```diff
@@ -216,10 +218,10 @@ exports[`authenticator-verification-webauthn renders form - webauthn supported 1
                   class="ods-4o5zYQ ods-76eppy ods-6lar65 ods-1OsrNt ods-2oc5SK ods-4XAbls ods-2NLKpK ods-1ll9ky ods-4IdjLs"
                 >
                   <span
-                    aria-label="Waiting for input"
```

Comment:
intentional change or just falls back to the default value?

Reply:
This is intentional, these "spinner loading" texts do not exist in v2 translation files. We cannot manually add one so I have to use plain english temporarily to prevent Loc I10 ERROR, I am creating a ticket and will be working with Alex, Deepa, and Translator to get new keys (like this) created for us.